airqo-platform/AirQo-frontend
# Showcase/ Tutorial for the mobile app features

#### Summary of Changes (What does this PR do?)
- Create a tutorial for first-time mobile app users to introduce the app's important features.
- For this specific PR, only the `home screen` is being focused on. The specific features on the home screen that feature in the showcase are `favorites`, `for you`, `KYA`, `air quality location cards` .

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Uninstall any version of the app first.
* Navigate to the `mobile folder` and Run the app using `flutter run`.
* Sign up using a new email or Log In and you'll be able to see the showcase.
* Review the code for any cleanups.

#### What are the relevant tickets?
- [MOB-117](https://airqoteam.atlassian.net/browse/MOB-117)

#### Screenshots (optional)
![1](https://user-images.githubusercontent.com/86492979/199234818-ce435e8b-2792-471b-a03f-f1ab84059750.jpg)

[MOB-117]: https://airqoteam.atlassian.net/browse/MOB-117?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
